discovery-provider/tests/test_get_challenges.py

```diff
@@ -0,0 +1,273 @@
+        Challenge(
+            id="trending_3", type=ChallengeType.trending, active=True, amount=5
+        ),
+    ]
+    user_challenges = [
+        # Finished the first challenge, disbursed
+        UserChallenge(
+            challenge_id="boolean_challenge_1",
+            user_id=1,
+            specifier="1",
+            is_complete=True,
+        ),
+        # Did finish the second challenge, did not disburse
+        UserChallenge(
+            challenge_id="boolean_challenge_2",
+            user_id=1,
+            specifier="1",
+            is_complete=True,
+        ),
+        # Did not finish challenge 3
+        UserChallenge(
+            challenge_id="boolean_challenge_3",
+            user_id=1,
+            specifier="1",
+            is_complete=False,
+        ),
+        # Inactive challenge
+        UserChallenge(
+            challenge_id="boolean_challenge_6",
+            user_id=1,
+            specifier="1",
+            is_complete=True,
+        ),
+        UserChallenge(
+            challenge_id="aggregate_challenge_1",
+            user_id=1,
+            specifier="1-2",  # compound specifiers, like if user1 invites user2
+            is_complete=True,
+        ),
+        # Ensure that a non-complete user-challenge isn't counted towards
+        # aggregate challenge score
+        UserChallenge(
+            challenge_id="aggregate_challenge_1",
+            user_id=1,
+            specifier="1-3",
+            is_complete=False,
+        ),
+        UserChallenge(
+            challenge_id="aggregate_challenge_2",
+            user_id=1,
+            specifier="1-2",
+            is_complete=True,
+        ),
+        UserChallenge(
+            challenge_id="aggregate_challenge_2",
+            user_id=1,
+            specifier="1-3",
+            is_complete=True,
+        ),
+        # Trending 1 should be finished and included
+        UserChallenge(
+            challenge_id="trending_1",
+            user_id=1,
+            specifier="06-01-2020",
+            is_complete=True,
+        ),
+        # Trending 2 should not be included
+        UserChallenge(
+            challenge_id="trending_2",
+            user_id=1,
+            specifier="06-01-2020",
+            is_complete=False,
+        ),
+    ]
+    disbursements = [
+        ChallengeDisbursement(
+            challenge_id="boolean_challenge_1",
+            user_id=1,
+            amount=5,
+            block_number=1,
+            specifier="1",
+        )
+    ]
+
+    # Wipe any existing challenges in the DB from running migrations, etc
+    session.query(Challenge).delete()
+    session.commit()
+    session.add_all(blocks)
+    session.commit()
+    session.add_all(challenges)
+    session.commit()
+    session.add_all(user_challenges)
+    session.commit()
+    session.add_all(disbursements)
+
+
+def test_get_challenges(app):
+    with app.app_context():
+        db = get_db()
+        with db.scoped_session() as session:
```

Comment:
i think this would be cleaner as a pytest fixture that can be responsible for setup & teardown https://stackoverflow.com/a/62784688/11435157

but fine ok w/ me the way it is since there's only one test so far

Reply:
Oh I see what you were saying. Yeah that would be nice - can move to this if we use the same setup later on.